sdk/keyvault/keyvault-keys/CHANGELOG.md

```diff
@@ -1,15 +1,20 @@
 # Release History
 
-## 4.8.0-beta.2 (Unreleased)
+## 4.8.0 (Unreleased)
 
 ### Features Added
 
+Since 4.7.2:
+- Added `hsmPlatform` property to `KeyProperties`.
```

Comment:
what does this parameter mean and why would a customer care it was added? Is it an input or output param? Etc. 

Maybe it's well documented in JSDoc below (I am not there yet)